tests/conftest.py

```diff
@@ -36,7 +47,42 @@ def pytest_internalerror(excinfo):
         raise excinfo.value
 
 
+def get_model_system(
+    type: str = 'original',
+    positions: List[List[float]] = [[0, 0, 0], [0.5, 0.5, 0.5]],
+    chemical_symbols: List[str] = ['Ga', 'As'],
+    orbital_symbols: List[List[str]] = [['s'], ['px', 'py']],
```

Comment:
I also am in the habit of using plural for lists or tuples of data. Given that this is a list of lists, I'd use a double plural, i.e. *orbital**s**_symbols*.

Actually, `symbols` is redundant: you even drop it in your `for`-loop later on.
Maybe use `atoms_orbitals`, which contains `atom_orbitals` that in turn contain `atom_orbital`?

(Btw, this is just a suggestion. I'm not going to argue about variable names, they just have to be understandable enough. In that sense, this here suffices.)